packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Beautiful test file. I could scan the whole file in 1 minute and understand what was happening!